scim2.yaml

```diff
@@ -1353,6 +1353,16 @@ paths:
           description: The order in which the "sortBy" parameter is applied.
           required: false
           type: string
+        - name: attributes
+          in: query
+          description: SCIM defined attributes parameter.
```

Comment:
let's avoid saying it as SCIM defined as the endpoint is not defined in the spec

Reply:
Fixed